src/config/test.yml

```diff
@@ -27,6 +27,7 @@ treeherder:
 
 try:
+  errorTaskUrl: "{{{host}}}{{{path}}}/raw-file/{{revision}}/error.yml"
```

Comment:
^ so this config file test.yml is _only_ used in tests (for mozilla-taskcluster)... I actually hardcoded the error graph to something on github until I land the bits to mozilla-central onward.